src/components/templates/SimpleTableSelect/index.ts

```diff
@@ -183,7 +183,7 @@ class SimpleTableSelect extends BridgeBase {
       parameters: this.parameters
     };
 
-    const filters = this.thePConn.getRawMetadata().config.promotedFilters ?? [];
+    const filters = (this.thePConn.getRawMetadata() as any).config.promotedFilters ?? [];
```

Comment:
Replace unnecessary cast to any with // @ts-ignore - promotedFilters missing from types